packages/neuron-ui/src/locales/en.json

```diff
@@ -1234,6 +1234,10 @@
         "tip": "Due to insufficient disk space, synchronization has been stopped. <br /> Please allocate more disk space or migrate the data to another disk.",
         "continue-sync": "Continue Sync",
         "migrate-data": "Migrate Data"
+      },
+      "import-exist-xpubkey-dialog": {
+        "replace-tip": "The watch wallet has been imported before, would you replace it?",
```

Comment:
`wallet wallet` should be `watch-only wallet`

Ref: https://river.com/learn/terms/w/watch-only-bitcoin-wallet/